src/Utils/util.ts

```diff
@@ -158,4 +159,72 @@ export const returnStringWhenError = (s: string) => {
             return s
         }
     }
+}
+
+export const setEntityActionDisplay = (action: CLM.ActionBase, entities: CLM.EntityBase[]): [string, string] => {
+    let name = `MISSING ENTITY`
+    let value = `MISSING VALUE`
+
+    const entity = entities.find(e => e.entityId === action.entityId)
+    if (entity) {
+        name = entity.entityName
+        if (entity.entityType !== CLM.EntityType.ENUM) {
+            value = `Entity Is Not Enum!`
```

Comment:
Do we need to internationalize this?

Reply:
Yea, we should although I wanted to keep the Util simple and there many other places that use "MISSING ACTION" and I was following that pattern for "MISSING ENTITY"

Given that we made entities used in SET_ENTITY actions non-deletable this case should never happen but we had to be defensive against it for future.

I guess we can make item in backlog to do all of these.